app/services/apiServices.js

```diff
@@ -35,11 +35,11 @@ export async function get(url, params, argOptions = {}) {
 }
 
 export async function post(url, params, argOptions = {}) {
-    var apiUrl = url
-    myHeaders.set('Content-Type', 'application/x-www-form-urlencoded')
+    var apiUrl = _config.urls.base + url
+    myHeaders.set('Content-Type', 'application/json')
```

Comment:
It applies on all the requests? We need the json content-type only for the new requests